ccd-definition/AuthorisationCaseField/system-update.json

```diff
@@ -187,5 +187,12 @@
     "CaseFieldID": "documentToBeSent",
     "UserRole": "caseworker-publiclaw-systemupdate",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "sendToCtsc",
+    "UserRole": "caseworker-publiclaw-systemupdate",
+    "CRUD": "R"
```

Comment:
does system update user need permissions for this at all? Do we ever send notifications from this user?

Reply:
No let's remove.